README.rst

```diff
@@ -59,6 +59,33 @@ There is one special method which allows transfer file between hosts.
         h2, "/path/to/file/on/h2/or/target/dir",
     )
 
+You can also mount devices.
+
+.. code:: python
+
+    from rrmngmnt import Host, RootUser
+    from rrmngmnt.filesystem import MountPoint
```

Comment:
I would avoid creating MountPoint class directly.
ppl should get instance via your method h.fs.mount_point(...) call, in my opinion.